src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileDownloadHandlerWidget.java

```diff
@@ -100,18 +93,20 @@ public void onFailure(Throwable caught) {
           public void onSuccess(
             RestrictionInformationResponse restrictionInformation
           ) {
-            configure(bundle, restrictionInformation);
+            configure(actionMenu, bundle, restrictionInformation);
           }
         }
       );
     }
   }
 
   public void configure(
+    final EntityActionMenu actionMenu,
     EntityBundle bundle,
     RestrictionInformationResponse restrictionInformation
   ) {
     view.clear();
+    this.view.setActionMenu(actionMenu);
```

Comment:
This is funky-a lot of the business logic is actually in the view. Should refactor, but for now, just pass the action menu along...